### PR TITLE
Update: WestWind.MarkdownMonster v2.0.5.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

image

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22561)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
Another one of these..

File: manifests/w/WestWind/MarkdownMonster/2.0.5.0/WestWind.MarkdownMonster.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Platform:
+- Windows.Desktop
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
+  Scope: user
```
Comment (vedantmgoyal2009) [inline manifests/w/WestWind/MarkdownMonster/2.0.5.0/WestWind.MarkdownMonster.installer.yaml:15]:
```suggestion
Installers:
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: inno
  Scope: machine
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
Can you try changing the Scope?

**Comment by OfficialEsco** [comment]
@palenshus can you take a look at this one as it passes Sandbox?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**